lib/modules/reducer.js

```diff
@@ -35,6 +46,7 @@ export default (state=DEFAULT, action={}) => {
         currentPage: pageData,
       };
     }
+
```

Comment:
I'll look into it. As a personal preference, I like to give these case statements a little breathing room. It helps to mentally separate them in the case their contents have newline spaces as well.